test/core/TilingSprite.js

```diff
@@ -25,6 +25,95 @@ describe('PIXI.TilingSprite', function ()
         });
     });
 
+    describe('.getLocalBounds()', function ()
+    {
+        before(function ()
+        {
+            this.tileSprite = new PIXI.extras.TilingSprite(PIXI.Texture.EMPTY);
+        });
+
+        beforeEach(function ()
+        {
+            sinon.stub(PIXI.Sprite.prototype, 'getLocalBounds');
+            this.tileSprite._bounds = {
+                getRectangle: sinon.spy(),
+            };
+            this.tileSprite._width = 1;
+            this.tileSprite._height = 2;
+            this.tileSprite._anchor = {
+                _x: 3,
+                _y: 4,
+            };
```

Comment:
In general, I'm not really a fan how anti-pattern the code in your tests are. Why are you not doing `this.tilingSprite.anchor.set(3, 4)` here or setting width and height in the TilingSprite constructor in `before`? It's not good form to set private variable, similar to my comment in your last PR.